### ISSUE TITLE
Write shop greeting messages

### ISSUE THREAD

**Issue by tagniam** [tagniam/Turn#63]
Every good RPG has shops with random greeting messages. This RPG is no different.

Jeremy's Tools Store needs random greeting messages to display whenever the player enters the store. Print the greeting right after the shop title.

For example:
```
 Jeremy's Tools Store
 "Buy somethin', would ya?"

 ----ITEM----------------COST------ 

 1) Arrows x 5		 200
 2) Bomb		 100
```

Make as many greetings as you'd like. Make sure they add a little personality to the game!

**Comment by Mego** [comment]
Working on this!

**Comment by tagniam** [comment]
Closed by #65
